### PR TITLE
log additionalProperties invalid type error only

### PR DESCRIPTION
**Pull request by raych1**

### CODE REVIEW

File: lib/liveValidation/operationValidator.ts [hunk 1]
```diff
@@ -276,17 +299,32 @@ export const schemaValidateIssueToLiveValidationIssue = (
...
-      const isMissingRequiredProperty = issue.code === "OBJECT_MISSING_REQUIRED_PROPERTY";
-      const isBodyIssue = path.startsWith(".body");
-
       if (issue.code === "MISSING_RESOURCE_ID") {
         // ignore this error for sub level resources
         if (path.includes("properties")) {
           skipIssue = true;
           return "";
         }
+      } else if (issue.code === "INVALID_TYPE" && isArmCall !== undefined && isArmCall === false) {
```
Comment (jianyexi) [inline lib/liveValidation/operationValidator.ts]:
```suggestion
      } else if (issue.code === "INVALID_TYPE" && isArmCall === false) {
```

File: lib/liveValidation/operationValidator.ts [hunk 2]
```diff
@@ -262,7 +282,10 @@ export const schemaValidateIssueToLiveValidationIssue = (
   input: SchemaValidateIssue[],
   operation: Operation,
   ctx: SchemaValidateContext,
-  output: LiveValidationIssue[]
+  output: LiveValidationIssue[],
+  info: OperationContext,
```
Comment (jianyexi) [inline lib/liveValidation/operationValidator.ts]: seem the 'info' here isn't specific, could you rename it ?

File: lib/liveValidation/operationValidator.ts (hunk 1 above)
Comment (xiaoxuqi-ms) [reply lib/liveValidation/operationValidator.ts]: this config is optional,

File: lib/liveValidation/operationValidator.ts [hunk 3]
```diff
@@ -276,17 +299,32 @@ export const schemaValidateIssueToLiveValidationIssue = (
...
-      const isBodyIssue = path.startsWith(".body");
-
       if (issue.code === "MISSING_RESOURCE_ID") {
         // ignore this error for sub level resources
         if (path.includes("properties")) {
           skipIssue = true;
           return "";
         }
+      } else if (issue.code === "INVALID_TYPE" && isArmCall !== undefined && isArmCall === false) {
+        if (issue.schemaPath.includes("additionalProperties")) {
```
Comment (xiaoxuqi-ms) [inline lib/liveValidation/operationValidator.ts:309]: what is the rule for additionalProperties, why we can ignore the invalid_type for additionalProperties?

File: lib/liveValidation/operationValidator.ts (hunk 2 above)
Comment (raych1) [reply lib/liveValidation/operationValidator.ts]: Renamed.

File: lib/liveValidation/operationValidator.ts (hunk 3 above)
Comment (raych1) [reply lib/liveValidation/operationValidator.ts:309]: This work is what we discussed in last two sync up meeting. The context is: oav new version added the `additionalProperties` type validation which appears to be a breaking change to service team. There's no easy approach to identify the service who has inconsistence between annotations of `additionalProperties` and API implementation. 

We don't want to introduce LSI by this new version release.  That's why we have this PR. Feel free to ping me offline for further questions.

**Review by xiaoxuqi-ms — APPROVED** [review]

**Pull request merged**